shared/src/commonMain/kotlin/org/hyperskill/app/notifications_onboarding/presentation/NotificationsOnboardingFeature.kt

```diff
@@ -1,26 +1,50 @@
 package org.hyperskill.app.notifications_onboarding.presentation
 
 import org.hyperskill.app.analytic.domain.model.AnalyticEvent
+import org.hyperskill.app.notification.local.cache.NotificationCacheKeyValues
 
 object NotificationsOnboardingFeature {
-    object State
+    internal data class State(val dailyStudyRemindersStartHour: Int)
+
+    data class ViewState(
+        val dailyStudyRemindersStartHour: Int,
```

Comment:
Why do we need `dailyStudyRemindersStartHour` in the viewState?

Reply:
This should be in `NotificationsOnboardingFeature.Action.ViewAction.ShowDailyStudyRemindersIntervalStartHourPickerModal` to select initial / current row in picker:

![Simulator Screenshot - iPhone 15 Pro - 2023-12-15 at 11 17 57](https://github.com/hyperskill/mobile-app/assets/8586063/91b86678-e57f-4c1d-ae46-3a96dbb8b1e4)